tests/unit/gapic/iot_v1/test_device_manager.py

```diff
@@ -15,10 +15,10 @@
 #
 import os
 import mock
-import packaging.version
 
 import grpc
 from grpc.experimental import aio
+import json
```

Comment:
Adding imports seems pointless (and lint-breaking) without also adding code which uses them.